### PR TITLE
Fixed - Pagination not working when using different permalink structure.

### PR DESCRIPTION
**Pull request by RajkiranBagal**
### Description
Fixed - Pagination not working when using different permalink structures.
### Screenshots
https://share.getcloudapp.com/xQuWvgLp

### Types of changes

 Bug fix (non-breaking change which fixes an issue) 

### How has this been tested?
Check by changing the different Permalink structure if pagination works

### Checklist:
- [x] My code is tested
- [x] My code follows accessibility standards 
- [x] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [x] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: dist/blocks/post/index.php [hunk 1]
```diff
@@ -313,6 +316,65 @@ function uagb_render_pagination( $query, $attributes ) {
...
+
+	return $base;
+}
+/**
+ * Returns the Paged Format.
+ * @param string $permalink_structure Premalink Structure.
+ * @param string $base Base.
+ * @since x.x.x
+ */
+function paged_format( $permalink_structure, $base ) {
```
Comment (vrundakansara) [inline dist/blocks/post/index.php:366]: Prefix these functions. Or add them in Helper class.

File: dist/blocks/post/index.php [hunk 2]
```diff
@@ -313,6 +316,65 @@ function uagb_render_pagination( $query, $attributes ) {
 
 	return wp_kses_post( implode( PHP_EOL, $links ) );
 }
+/**
+ * Builds the base url.
+ * @param string $permalink_structure Premalink Structure.
+ * @param string $base Base.
+ * @since x.x.x
+ */
+function build_base_url( $permalink_structure, $base ) {
```
Comment (vrundakansara) [inline dist/blocks/post/index.php:330]: Prefix these functions. Or add them in Helper class.

Comment (RajkiranBagal) [reply dist/blocks/post/index.php]: Done

File: dist/blocks/post/index.php (hunk 1 above)
Comment (RajkiranBagal) [reply dist/blocks/post/index.php]: Done

**Pull request merged**